lib/poller.js

```diff
@@ -150,4 +150,8 @@ function setup (__settings = null, schemasToAdd = [], schemasToRemove = []) {
   return initializeEachSchema(schemasToAdd)
 }
 
-module.exports = { setup, reload }
+function getActiveSchemas () {
+  return Array.from(schemaCallbacks.keys())
```

Comment:
Couldn't this be just `return _.keys(schemaCallbacks)`?

Reply:
Since it's a Map lodash doesn't work on it. And I want to return an array not a map iterator so I also have to use Array.from